app/src/main/java/org/mozilla/fenix/search/SearchController.kt

```diff
@@ -58,6 +58,13 @@ class DefaultSearchController(
             }
 
             context.metrics.track(event)
+            if (CustomSearchEngineStore.isCustomSearchEngine(
+                    context,
+                    store.state.searchEngineSource.searchEngine.identifier
+                )
+            ) {
+                context.components.analytics.metrics.track(Event.SearchWithCustomEngine)
```

Comment:
How is this probe different form the event in `createSearchEvent` above? It looks like it also sends whether or not a search is custom or not https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/search/SearchController.kt#L130

Reply:
From what I understand, `createSearchEvent` creates a `Event.PerformedSearch` which records:
 - a [search event](https://github.com/mozilla-mobile/fenix/blob/8aa04c048e9df949b0be31bbc9c079fb57be2c77/app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt#L103) that can have only [these](https://github.com/mozilla-mobile/fenix/blob/8aa04c048e9df949b0be31bbc9c079fb57be2c77/app/metrics.yaml#L62) values (no 'custom' here).
- a [counter](https://github.com/mozilla-mobile/fenix/blob/8aa04c048e9df949b0be31bbc9c079fb57be2c77/app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt#L102) with the labels `<search-engine-name>.<source>` (Description for `search_count` [here](https://github.com/mozilla-mobile/fenix/blob/8aa04c048e9df949b0be31bbc9c079fb57be2c77/app/metrics.yaml#L322)). So this counter is the place where "custom" is used. 

@boek is this correct?

Is this counter enough for what we need here? 
If so I will remove my event `Event.SearchWithCustomEngine`(this event only tracks if the user performed a search with a custom search engine).
How should i proceed?

Reply:
Yes this is correct. We can probably remove `SearchWithCustomEngine` :)